src/__snapshots__/App.test.js.snap

```diff
@@ -0,0 +1,30 @@
+  </p>
+</div>
+`;
+
+exports[`All components should match QUOTE snapshot 1`] = `
+<div
+  className="quotes"
+>
+  <h1>
+    I love this very motivational statement:
+  </h1>
+  <hr />
+  <p>
+    Loading...
+  </p>
```

Comment:
- [ ] It is good that you added a snapshot for the App component, but you missed out on other components. The project requires that you take a snapshot of **_ALL_** components. Kindly fix this

<img width="676" alt="image" src="https://github.com/noelfoka/math-magicians/assets/84156714/0a0f4048-5f0a-4111-ac15-2a2c93d56c49">

- [ ] Based on the project requirement to take snapshots for each component tested, it is best to place each test in a separate file and generate corresponding `.snap` files, here's how you can organize your tests:

1. Create a file named `Home.test.js` for testing the `Home` component:

```javascript
import React from 'react';
import { render, screen } from '@testing-library/react';
import Home from './Home'; // Adjust the import path

describe('Home component', () => {
  it('should render Home component', () => {
    render(<Home />);
    const userElement = screen.getByTestId('user-id');
    expect(userElement).toBeInTheDocument();
    expect(userElement).toHaveTextContent('History');
  });
});
```

2. Create a file named `Calculator.test.js` for testing the `Calculator` component:

```javascript
import React from 'react';
import { render, screen } from '@testing-library/react';
import Calculator from './Calculator'; // Adjust the import path

describe('Calculator component', () => {
  it('should render Calculator component', () => {
    render(<Calculator />);
    screen.getAllByRole('button');
  });
});
```

3. Create a file named `Quote.test.js` for testing the `Quote` component:

```javascript
import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote'; // Adjust the import path

describe('Quote component', () => {
  it('should match QUOTE snapshot', () => {
    const quote = renderer.create(<Quote />).toJSON();
    expect(quote).toMatchSnapshot();
  });
});
```

Make sure to adjust the import paths according to your project structure.

To generate `.snap` files for these tests, you'll need to run your tests using a testing framework like Jest. When you run your tests, Jest will automatically generate snapshot files in the `__snapshots__` directory next to your test files. The generated snapshot files will have names like `<ComponentName>.test.js.snap` and will contain the serialized representations of your component's rendered output.